### ISSUE TITLE
Backing Up Drafts Automatically

### ISSUE THREAD

**Issue by stupoole** [SublimeText/LaTeXTools#1280]
Hi there,

I have been wading through code I know nothing about trying to find the line that makes the PDF in order to attempt to add a new line which would save a copy of the pdf to a "./drafts" folder with the name "texfilename - YYMMDDHHMM.pdf". I'm using ST3 and latextools with a miktex 2.9 install.

Please let me know if anyone has a way to do this using latextools' already available optioins. If not, I'd like to find a way to do so.

Thanks,
Stu

**Comment by r-stein** [comment]
I don't think that we support something like this. Creating a backup for every build will lead to a huge draft folder. I usually use git manage the tex files. Nonetheless one could write a script to backup or pdf.
